### DISCUSSION TITLE
Simple question about MEEP units and epsilon

### DISCUSSION THREAD

**Discussion by 10UAikawa** [NanoComp/meep#2672] [Q&A]
So MEEP uses dimensionless units. So, if I have a quartz material that has a relative permittivity that is 4.41 F/m to convert that into dimensionless units I would have to divide that value by the vacuum permittivity of 8.854E-12 F/m.

So, if I do the conversion to get dimensionless units 4.41/8.854E-12 = 498079963858. This is an extremely large number and I'm not sure if this is the correct way to go about it. 

Quartz_material = mp.Medium(epsilon=498079963858)

Any help is appreciated.

**Comment by stevengj** [comment]
> relative permittivity that is 4.41 F/m 

*Relative* permittivity εᵣ is dimensionless — it does not have units of F/m, because it has *already* been divided by ε₀.  (This is different from the absolute permittivity ε: εᵣ = ε/ε₀.)

The `epsilon` in Meep *is* the relative permittivity εᵣ, no additional computation required.   Notationally, we simply drop the "r" subscript (a common practice among authors dealing with dimensionless units in EM).

**Reply by 10UAikawa** [reply]
Thanks for the clarification!
